protostuff-parser/src/main/antlr3/io/protostuff/parser/ProtoParser.g

```diff
@@ -113,6 +116,12 @@ annotation_keyval [Proto proto, Annotation annotation]
         )
     ;
 
+doc_entry [Proto proto]
+    :   DOC {
+            proto.addDoc($DOC.text.substring(3).trim());
```

Comment:
I think it is better to remove `.trim()`: it removes all whitespaces even if we want to keep them. 
For example, code blocks:

```
/// Message description
/// Usage:
/// <code>
/// class X extends Foo<Bar> {
///     public static final int BAZ = 1;
///     int z() {
///         // ...
///     }
/// }
/// </code>
```

This way, `$DOC.text.substring(3)` should be enough.
Later we can add custom formats like `TRIM`, `REMOVE_LEADING_SPACE`.

```
docs(container) ::= <<
<if(container.docs)>

/**
<container.docs:{doc |  * <doc; format="TRIM">}; separator="\n">
 */
<endif>
>>
```